src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -357,15 +361,31 @@ private void SaveSettingsCollectionStylesCss()
 			}
 		}
 
+		private void AddRtlCreditsCssRules(StringBuilder sb)
```

Comment:
I see that previously this was already being done, but it's really inexcusable to be putting static css in c#, I'm sure you'll agree. Put in a class, use a .less file to trigger on that class.